src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -37,6 +38,15 @@
 
     private Map<String, String> qualifiers = new Hashtable<>();
 
+    // List of charts that can be optionally excluded from being rendered.
+    // Default behavior is to render all charts.
+    private final List<ChartType> chartsToRender = new ArrayList<>(
```

Comment:
Set or List?

Reply:
Currently this is a List. Do you have a preference? One over the other?

Reply:
Does the order matter for you? If not, use Set

Reply:
I feel a List is appropriate